src/connection/host_lookup.c

```diff
@@ -298,42 +299,40 @@ load_hosts_file ()
 	  strcpy (temp_token, token);
 	  if (hostent_flag == INSERT_IPADDR)
 	    {
-	      if (ip_format_check (temp_token) == false)
+	      if (is_valid_ip (temp_token) == false)
 		{
 		  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_UHOST_INVALID_FORMAT, 4, "IP address", token, line_num,
-			  USER_HOSTS_FILE);
+			  CUBRID_HOSTS);
 
-		  user_host_Map.clear ();
-		  fclose (fp);
-		  goto load_fail_phase;
+		  continue;
 		}
 	      else
 		{
 		  strncpy (ipaddr, token, IPADDR_LEN);
 		  ipaddr[IPADDR_LEN - 1] = '\0';
-
 		  hostent_flag = INSERT_HOSTNAME;
 		}
 	    }
 	  else
 	    {
-	      if (strlen (token) > HOSTNAME_LEN - 1)
+	      str_len = strlen (temp_token);
+	      if (str_len > HOSTNAME_LEN - 1)
 		{
-		  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_UHOST_HOST_NAME_TOO_LONG, 3, token, line_num,
```

Comment:
Should I roll back this coded?
Because user cannot know which one is the problem.
@kisoo-han

Reply:
Ok, let's keep change.

Reply:
Ok. I will recover it.

Reply:
Considering compatibility with glibc, we don't need to give these error messages at library level. If necessary, it is better to do it in the function that call host lookup.
So, do not roll back.

Reply:
It does not print error into the stdout. Just record in the log. 
If you mean that log recording is also banned. I will keep it.